.github/workflows/build-browser-target.yml

```
@@ -0,0 +1,39 @@
+name: Build Browser on PR Target
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches-ignore:
+      - 'l10n_master'
+      - 'cf-pages'
+    paths:
+      - 'apps/browser/**'
+      - 'libs/**'
+      - '*'
+      - '!*.md'
+      - '!*.txt'
+  workflow_call:
+    inputs: {}
+  workflow_dispatch:
+    inputs:
+      sdk_branch:
+        description: "Custom SDK branch"
+        required: false
+        type: string
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  check-run:
+    name: Check PR run
+    uses: bitwarden/gh-actions/.github/workflows/check-run.yml@main
+
+  run-workflow:
+    name: Run Build Browser on PR Target
+    needs: check-run
+    if: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
+    uses: ./.github/workflows/build-browser.yml
+    secrets: inherit
```

Should each of the `pull_request` workflows be passing the inputs along?

```suggestion
    name: Run Build Browser on PR Target
    needs: check-run
    if: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
    uses: ./.github/workflows/build-browser.yml
    secrets: inherit
    with:
      sdk_branch: ${{ inputs.sdk_branch }}
```

---

Hmmm, good point, it's hard when you can't test these things 🙃

---

Fixed by removing workflow_dispatch!